detect_secrets/plugins/high_entropy_strings.py

```diff
@@ -110,6 +128,44 @@ def _analyze_ini_file(self, file, filename):
 
         return potential_secrets
 
+    def _analyze_yaml_file(self, file, filename):
+        """
+        :returns: same format as super().analyze()
```

Comment:
Nit: Maybe add e.g. `:raises: yaml.YAMLError` and the same w/ `configparser.Error` to `_analyze_ini_file`.